pom.data.xml

```diff
@@ -543,7 +543,9 @@
   <modules>
     <module>./sdk/batch/microsoft-azure-batch</module>
     <module>./sdk/cosmos</module>
```

Comment:
@mitchdenny Do I need to bump the version of pom.data.xml because I updated the modules?